bin/s2nc.c

```diff
@@ -62,7 +62,9 @@ void usage()
     fprintf(stderr, "  -i,--insecure\n");
     fprintf(stderr, "    Turns off certification validation altogether.\n");
     fprintf(stderr, "  -r,--reconnect\n");
-    fprintf(stderr, "    Drop and re-make the connection with the same Session-ID\n");
+    fprintf(stderr, "    Drop and re-make the connection using Session ticket. If session ticket is disabled, then re-make the connection using Session-ID \n");
+    fprintf(stderr, "  -T,--no-session-tickets \n");
+    fprintf(stderr, "    Do not support session tickets for resumption.\n");
```

Comment:
Nit: Can you please update the usage message to "Disable session tickets for resumption" ?